.gitignore

```diff
@@ -0,0 +1,39 @@
+# IntelliJ
+.idea/
+
+# Gradle
+.gradle/
+*.iml
+build
+out
+config.properties
+
+# Compiled class file
+*.class
+
+# Log file
+*.log
+
+# BlueJ files
+*.ctxt
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
+# Package Files #
+*.jar
+*.war
+*.nar
+*.ear
+*.zip
+*.tar.gz
+*.rar
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+
+#reports
+/target/*
+
+#Log4J
+/log/*
```

Comment:
Dear raul please put salt of the line in this site.

Reply:
thank you, this wrong was fixed